legacy/toml-config/src/main/java/io/zeebe/legacy/tomlconfig/LegacyConfigurationSupport.java

```diff
@@ -0,0 +1,257 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.legacy.tomlconfig;
+
+import io.zeebe.legacy.tomlconfig.util.Loggers;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.function.Function;
+
+public class LegacyConfigurationSupport {
+  private static final Map<String, Replacement> MAPPING_GATEWAY = new HashMap<>();
+  private static final Map<String, Function<String, String>> VALUE_CONVERTERS_GATEWAY =
```

Comment:
```suggestion
  private static final Map<String, UnaryOperator<String>> VALUE_CONVERTERS_GATEWAY =
```

Note: I guess you can't directly commit this change as it will need to import `UnaryOperator` first.

Reply:
done